features/facebookImport/src/model/analyses/report/unknown-top-level-folders-analysis.js

```diff
@@ -2,17 +2,14 @@ import React from "react";
 import BasicList from "../../../components/basicList/basicList.jsx";
 import ReportAnalysis from "./report-analysis.js";
 import topFolderNames from "../../../static/topFolders.js";
-import {
-    relevantZipEntries,
-    removeEntryPrefix,
-} from "../../importers/utils/importer-util.js";
+import { relevantZipEntries } from "../../importers/utils/importer-util.js";
 
 async function extractTopLevelFolderNamesFromZip(zipFile) {
     const relevantEntries = await relevantZipEntries(zipFile);
     const topLevelFolderNames = new Set();
 
-    relevantEntries.forEach((filename) => {
-        const noIdFileName = removeEntryPrefix(filename);
+    relevantEntries.forEach((entry) => {
+        const noIdFileName = entry._path;
```

Comment:
Same as above

Reply:
Not sure what was _above_, but seeing how `noIdFileName` is only being used once, and naming-wise makes less sense now, I'd just inline `entry._path` below.

Reply:
Now I can see what was above - basically what I said, so could have just given a thumbs up :D